src/components/Card.js

```diff
@@ -1,10 +1,19 @@
 import { useState } from "react";
 
 import "./Card.css";
+import { AiFillEdit } from "react-icons/ai";
+import { AiFillCheckCircle } from "react-icons/ai";
+import { AiFillCloseCircle } from "react-icons/ai";
 
 function Card(props) {
-  let classNames = require('classnames');
+  let classNames = require("classnames");
+
   const [checked, setChecked] = useState(props.checked);
+  const [editable, setEditable] = useState(false);
+  const [caption, setCaption] = useState(props.caption);
+  const [curCaptionValue, setCurCaptionValue] = useState(caption);
+  const [text, setText] = useState(props.text);
+  const [curTextValue, setCurTextValue] = useState(text);
```

Comment:
And then get rid of states, then duplicates fields of props. You can use parent's callback instead